src/modules/imports/import_string.rs

```diff
@@ -9,10 +9,8 @@ pub struct ImportString {
 
 impl ImportString {
     fn resolve_path(&mut self, meta: &ParserMetadata, tok: Option<Token>) -> SyntaxResult {
-        if self.value == "std" {
-            self.value = "[standard library]".to_string();
+        if self.value.starts_with("std") {
```

Comment:
The same reason as before

```suggestion
        if self.value.starts_with("std/") {
```